runtime/src/message_processor.rs

```diff
@@ -180,7 +180,7 @@ impl PreAccount {
             }
             if !is_writable // line coverage used to get branch coverage
                 || pre.executable()
-                || program_id != pre.owner()
+                || program_id != post.owner()
```

Comment:
Does this need gated?

Reply:
n/m. I see #18492 now :wink:

Reply:
Ah, totally missed that in review. Needs gating because you could set executable to be true and get a successful transaction right?

Reply:
Yup, realized that as soon as I hit merge ;-). 

This covers it: https://github.com/solana-labs/solana/pull/18492

Reply:
> Ah, totally missed that in review. Needs gating because you could set executable to be true and get a successful transaction right?

These get easier to see the more cluster deaths you cause :slightly_smiling_face: